test/test_unit_client.py

```diff
@@ -286,7 +286,7 @@ def test_client_passes_through_api_get_variable_params(self, mock_api_instance,
             mock_api_config):
         Client().get('variable_id')
 
-        mock_api_instance.return_value.get_variable.assert_called_once_with('variable_id')
+        mock_api_instance.return_value.get_variable.assert_called_once_with('variable_id', None)
```

Comment:
why giving None if its default parameter?

Reply:
it is just asserting that None is being passed. Notice 
`Client().get('variable_id')` is what is being called which references `get(self, variable_id, version=None):`
and here that none is verifying that the internal call `self._api.get_variable` is given None
```
    def get(self, variable_id, version=None):
        """
        Gets a variable value based on its ID
        """
        return self._api.get_variable(variable_id, version)
```